src/libraries/System.Console/src/System/ConsolePal.WebAssembly.cs

```diff
@@ -163,7 +168,15 @@ public static string Title
             char sourceChar, ConsoleColor sourceForeColor,
             ConsoleColor sourceBackColor) => throw new PlatformNotSupportedException();
 
-        public static void Clear() => throw new PlatformNotSupportedException();
+        public static void Clear()
+        {
+            if (!s_consoleInitialized)
+            {
+                s_console = (JSObject)System.Runtime.InteropServices.JavaScript.Runtime.GetGlobalObject("console");
```

Comment:
s_consoleInitialized is needed because GetGlobalObject("console") may return null?

Reply:
Exactly.